helix-view/src/theme.rs

```diff
@@ -157,6 +169,10 @@ impl Theme {
         self.highlights[index]
     }
 
+    pub fn name(&self) -> String {
+        self.name.to_string()
+    }
```

Comment:
Should return `&str`

Reply:
Done, but I'm still converting that value to a string to avoid rust lifetime issues wrt the context. This method is only going to be called infrequently so conversion to a string seems reasonable to me.

Reply:
It's more colloquial with the rest of the codebase. It's better to leave it to the consumer of `name` to decide when to clone. `String` is ok with the current `cx.editor.set_status` since it needs it for lifetimes as you say but if instead the caller used `format!("The theme is {}", theme.name())`, you would be re-allocating it twice rather than once.